emperor/support_files/js/controller.js

```diff
@@ -45,6 +63,10 @@ EmperorController = function(dm, divId){
   this.buildUI();
 };
 
+/**
+ * Helper method to add additional scenePlotViews (i.e. another plot)
```

Comment:
I think the way to write this is ScenePlotViews as that's the name of the class.